go.mod

```diff
@@ -1,12 +1,15 @@
 module github.com/Cray-HPE/hms-power-control
 
 go 1.16
-	//todo hms-base needs to be converted to hms-xname as soon as that package is available.
+
+//todo hms-base needs to be converted to hms-xname as soon as that package is available.
 
 require (
 	github.com/Cray-HPE/hms-base v1.15.1
```

Comment:
We should updated the hms-base package to v2.0.0, so we get all of our xname related types from hms-xname.

Reply:
agree; I will do this in our integration.